src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -444,15 +444,15 @@ const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { nam
       }),
       h(JupyterFrameManager, {
         frameRef,
-        onClose: () => Nav.goToPath('workspace-notebooks', { namespace, name }),
-        details: { notebookName, name, namespace }
+        onClose: () => Nav.goToPath('workspace-notebooks', { googleProject, name }),
+        details: { notebookName, name, googleProject }
```

Comment:
These details would probably need to contain `googleProject` _and_ `namespace` if both are used in `JupyterFrameManager`.